src/cap-description.adoc

```diff
@@ -194,13 +194,13 @@ Quadrant 1 encodes permissions for executable capabilities and the <<m_bit>>.
 | 4-5   | ✔ | ✔ | ✔ | ✔ | ✔ |     | Mode^1^  | Execute + Data & Cap RW
 | 6-7   | ✔ | ✔ |   |   | ✔ |     | Mode^1^  | Execute + Data RW
 9+| *Quadrant 2: Restricted capability data read/write*
-9+| bit[2] - write. R and C implicitly granted, LM dependent on W permission.
```

Comment:
This is also true, see #355 although right now all of the writable encodings are reserved.

Reply:
ok - I added a comment about that